modules/react/layout/lib/Flex.tsx

```diff
@@ -5,7 +5,7 @@ import {Box, BoxProps} from './Box';
 
 import {flex, FlexStyleProps} from './utils/flex';
 
-export type FlexProps = BoxProps & FlexStyleProps;
+export type FlexProps = Omit<BoxProps, 'display'> & FlexStyleProps;
```

Comment:
VSCode intellisense was getting confused with BoxProps display and not providing any helpful suggestions. So instead of merging Box's `display` with Flex's `display`, we're removing one and adding the other. The types remain the same, but VSCode works better.